docs/man/docker-inspect.1.md

```diff
@@ -30,69 +30,109 @@ each result.
 
 To get information on a container use it's ID or instance name:
```

Comment:
Nothing special, I was reading this man page to figure out how I can write this `-f` and didn't get the answer. I think we should add something, then I found some format issues and the outdated inspect out, so just change them at one time.
The example output don't need to be updated in time, but it's too old now, I think it's time to refresh. :)